### ISSUE TITLE
-text doesn't write into a file anymore

### ISSUE THREAD

**Issue by TheRealAgentK** [cflint/CFLint#519]
It seems that if you specify -text without -textfile, there's not default file (cflint-results.txt) being generated anymore as it used to previously. Instead the text output seems to be spit out to stdout.

I think that's new in the current release, I can replicate it in dev 1.4.0-snapshot.

Specifying -xml and -html without their respective file attributes creates the respective output files just fine.

**Comment by TheRealAgentK** [comment]
Maybe it's *not* new, but the behaviour in CFLintCLI.java is def. different for text than any other output format. For a start there's no default file name set, even though it's documented in the help. Pretty sure that'd be the core fix for the problem

```
 private String xmlOutFile = "cflint-result.xml";
    private String xmlstyle = CFLINT;
    private String htmlOutFile = "cflint-result.html";
    private String htmlStyle = "plain.xsl";
    private String jsonOutFile = "cflint-result.json";
    private String textOutFile = null;
```

Then further down, text is again treated differently than html/json etc:
```
     try (final Writer textwriter = stdOut || textOutFile == null ? new OutputStreamWriter(System.out)
                    : new FileWriter(textOutFile)) {
...
...
 try (final Writer htmlwriter = stdOut ? new OutputStreamWriter(System.out) : new FileWriter(htmlOutFile)) {
            ```

**Comment by TheRealAgentK** [comment]
Let me know what you think @ryaneberly @KamasamaK

**Comment by KamasamaK** [comment]
I agree that `cflint-results.txt` should be the default as the help indicates.

**Comment by TheRealAgentK** [comment]
Fixed it in PR #520 - please have a look and if @ryaneberly agrees to, lets merge it in.
